pkg/controllers/user/monitoring/metricExpression.go

```diff
@@ -2714,7 +2714,7 @@ metadata:
     metric: disk-wal-fsync-duration-seconds-sum-quantile
     source: rancher-monitoring
 spec:
-  expression: sum(histogram_quantile(0.99, rate(etcd_disk_wal_fsync_duration_seconds_bucket[5m])))
+  expression: histogram_quantile(0.99, sum(rate(etcd_disk_wal_fsync_duration_seconds_bucket[5m])))
     
```

Comment:
```sum(histogram_quantile(0.99, rate(etcd_disk_wal_fsync_duration_seconds_bucket[5m])))``` the sum is for sum value from mutiple nodes inside the cluster. This one ```histogram_quantile(0.99, sum(rate(etcd_disk_wal_fsync_duration_seconds_bucket[5m])))```  throw error no datapoint

Reply:
Yes, `sum(histogram_quantile(0.99, rate(etcd_disk_wal_fsync_duration_seconds_bucket[5m])))` is a bad expression, `histogram_quantile(0.99, sum(rate(etcd_disk_wal_fsync_duration_seconds_bucket[5m])))` is a wrong expression, I have committed the new one: `sum(histogram_quantile(0.99, sum(rate(etcd_disk_wal_fsync_duration_seconds_bucket[5m])) by (instance, le)))`